pkg/service/backup/service.go

```diff
@@ -76,9 +80,22 @@ func NewService(session gocqlx.Session, config Config, metrics metrics.BackupMet
 		scyllaClient:   scyllaClient,
 		clusterSession: clusterSession,
 		logger:         logger,
+
+		backoffConfiguration: DefaultStagesBackoffSetup(),
 	}, nil
 }
 
+func NewTestService(session gocqlx.Session, config Config, metrics metrics.BackupMetrics, clusterName ClusterNameFunc, scyllaClient scyllaclient.ProviderFunc,
```

Comment:
Future us and other maintainers will appreciate docs to distinguish between `NewService` and `NewTestService` ;)

Reply:
Done